stdlib/asyncio/tasks.pyi

```diff
@@ -41,7 +28,12 @@ ALL_COMPLETED: str
 def as_completed(
     fs: Iterable[_FutureT[_T]], *, loop: Optional[AbstractEventLoop] = ..., timeout: Optional[float] = ...
 ) -> Iterator[Future[_T]]: ...
-def ensure_future(coro_or_future: _FutureT[_T], *, loop: Optional[AbstractEventLoop] = ...) -> Future[_T]: ...
+@overload
+def ensure_future(coro_or_future: _FT, *, loop: Optional[AbstractEventLoop] = ...) -> _FT: ...  # type: ignore
+@overload
```

Comment:
This overload is redundant because Coroutine inherits from Awaitable.

Reply:
Good catch.